src/helpers/getUser.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from '../Types';
+
+export const getUser = (
```

Comment:
Use more descriptive name
```suggestion
export const getUserById = (
```